### ISSUE TITLE
No cell background colour in Phase_F.html

### ISSUE THREAD

**Issue by robintw** [DeepBlueCLtd/LegacyMan#525]
In data/Britain_Cmplx/Phase_F.html there are bgcolors specified for various table cells, eg:

```
<td bgcolor="#FFFF00">&nbsp;</td>
<td bgcolor="#0000FF">&nbsp;</td>
<td bgcolor="#FFFF00">&nbsp;</td>
<td bgcolor="#0000FF">&nbsp;</td>
```

These are not converted to any extra outputclass values in the resulting DITA.

@IanMayo can you tell me how you want these converted and I'll add the relevant code?

**Comment by IanMayo** [comment]
Thanks for that spotting that  @robintw 

These are the color codes found in the source content.  These should be new `outputclass` attributes in `htmlToDita` processing.

These should be `bkRed`, `bkYellow` and `blBlue`.  We'll introduce these as new styles in `template\Fi3ldMan\oxygen.css`.

Note: I'll investigate specifying a css style-sheet in the DITA-OT transform, so that the shading is visible in our `target/html` output.

**Comment by robintw** [comment]
I've fixed this now in #524 , but the styles don't seem to be in the CSS file so they're not being displayed yet.

**Comment by IanMayo** [comment]
Sure, job for me.

**Comment by IanMayo** [comment]
CSS styling also fixed in #524
